ui/webui/src/components/app.jsx

```diff
@@ -107,9 +112,31 @@ export const Application = () => {
         cockpit.location.go([newStep.id]);
     };
     const title = _("Anaconda Installer");
+    const prerelease = _("Pre-release");
 
     return (
-        <Page data-debug={conf.Anaconda.debug}>
+        <Page
+          data-debug={conf.Anaconda.debug}
+          additionalGroupedContent={
+              <PageSection variant={PageSectionVariants.light}>
+                  <Flex spaceItems={{ default: "spaceItemsSm" }} alignItems={{ default: "alignItemsCenter" }}>
+                      <TextContent>
+                          <Text component="h1">{title}</Text>
+                      </TextContent><br />
+                      <Popover
+                        aria-label=""
```

Comment:
empty aria label cancels the purpose of aria label :) See docs, says 'Accessible label, required when header is not present' 

You have defined the header here, so it's fine to drop it.